lib/go/thrift/application_exception.go

```diff
@@ -49,6 +50,7 @@ var defaultApplicationExceptionMessage = map[int32]string{
 	INVALID_TRANSFORM:              "Invalid transform",
 	INVALID_PROTOCOL:               "Invalid protocol",
 	UNSUPPORTED_CLIENT_TYPE:        "Unsupported client type",
+	VALIDATOR_CHECK_FAILED:         "Validator check failed",
```

Comment:
"validator check failed" sounds weird to me, maybe it should be "validator failed" or "validation failed"?